### ISSUE TITLE
cannot `cd ..` 

### ISSUE THREAD

**Issue by Carreau** [ipython/ipython#8983]
Seen on @njsmith linux laptop, Debian, IPython 4.0.0, Python 3.4.3

we can `cd ./..`, and any other cd command, but not just `cd ..`

From inside a venv. 

Maybe related, `hist -t` give nothing. 

explicitely calling  get_ipython().magic('cd ..')

apparently the cd magic, get only `.` as argument...

**Comment by njsmith** [comment]
A bit of print debugging reveals the following oddity:
```
(Pdb) p DollarFormatter().vformat("..", args=[], kwargs={})
'.'
```

**Comment by Carreau** [comment]
duplicate of #8919 i think.

**Comment by Carreau** [comment]
closed by #8985
